jdaviz/app.py

```diff
@@ -325,9 +325,9 @@ def __init__(self, configuration=None, *args, **kwargs):
         self.auto_link = kwargs.pop('auto_link', True)
 
         # Imviz linking
-        self._link_type = 'pixels'
+        self._align_by = 'pixels'
         if self.config == "imviz":
-            self._wcs_use_affine = None
+            self._fast_approximation = None
```

Comment:
would it be useful to keep the `wcs` prefix since its only applicable if `align_by == 'wcs'`?  (I'm ok either way, just a thought that might help prevent the need to go to docs)

Reply:
I like that idea, implemented.